src/components/Calculator/RefinancingCalculatorForm.tsx

```diff
@@ -47,6 +58,16 @@ export const RefinancingCalculatorForm = JSX<RefinancingCalculatorProp>(
       paramsViewSettings,
     );
 
+    const paymentTypeElement = renderPaymentType({
+      title: isMobile ? 'Тип платежа' : '',
+      orientation: 'vertical',
+      ...field('isAnnuity'),
+    });
+
+    useEffect(() => {
+      setContainerUpdateAt && setContainerUpdateAt(Date.now());
+    }, [calculatorParams.isExtraMoney]);
```

Comment:
Тоже не оч. красивый вариант. Но хоть что то. Будет в таком виде (без Date.now()) работать?
```suggestion
    }, [onContainerKeyChange, calculatorParams.isExtraMoney]);
```

Reply:
отказался от такого подхода, в пользу resizeObserver в Foldable